src/Sulu/Bundle/SecurityBundle/Domain/Event/UserCreatedEvent.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Domain\Event;
+
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+use Sulu\Bundle\SecurityBundle\Admin\SecurityAdmin;
+use Sulu\Component\Security\Authentication\UserInterface;
+
+class UserCreatedEvent extends DomainEvent
+{
+    /**
+     * @var UserInterface
+     */
+    private $user;
+
+    /**
+     * @var mixed[]|null
+     */
+    private $payload;
+
+    /**
+     * @param mixed[] $payload
+     */
+    public function __construct(UserInterface $user, array $payload)
+    {
+        parent::__construct();
+
+        $this->user = $user;
+        $this->payload = $payload;
+    }
+
+    public function getEventType(): string
```

Comment:
would provide a getter for `$this->user` if possible:
https://github.com/sulu/sulu/blob/2.x/src/Sulu/Bundle/CategoryBundle/Domain/Event/CategoryCreatedEvent.php#L50-L53

Reply:
I renamed the variable to `resourceUser` otherwise it conflicts with the `user` of the domainevent. But I'm not really happy with the naming, do you have something better in mind?

Reply:
you are right, have not thought about that 🙈  
`resourceUser` sounds good to me